site/src/docs/components/checkbox/customisation.mdx

```diff
@@ -12,57 +11,62 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Customisation
 
 <Notification type="alert" label="Checkbox customisation works differently!" className="siteNotification">
-  Checkbox is an older HDS component and therefore its customisation works slightly differently. Instead of the <code>theme</code> property, you need to use the <code>style</code> property. This will be fixed in a future update.
+  Checkbox is an older HDS component and therefore its customisation works slightly differently. Instead of the{' '}
```

Comment:
{' '} ? 😁

Reply:
It's weird thing that Prettier wanted to add! Probably because the sentence was so long that prettier wanted to insert a new line but make sure that the displayed text still has a space between the words where it was cut. It looks good on the site and here's the proof:
<img width="948" alt="Screenshot 2022-09-09 at 13 25 59" src="https://user-images.githubusercontent.com/108321134/189329656-6c58e659-db50-431d-8889-ec9a98b25ad1.png">

Reply:
I removed the spaces